openshift/scripts/common/envars

```diff
@@ -7,4 +7,5 @@ PROJ_PROD="${PROJ_PROD:-e1e498-prod}"
 TAG_PROD="${TAG_PROD:-prod}"
 PATH_BC="${PATH_BC:-$(dirname ${0})/../templates/build.bc.yaml}"
 PATH_DC="${PATH_DC:-$(dirname ${0})/../templates/deploy.dc.yaml}"
+PATH_NATS="${PATH_NATS:-$(dirname ${0})/../templates/nats.yaml}"
```

Comment:
We have ".bc.yml", ".dc.yml", ".config.yml" and ".cronjob.yml" - wonder if we need a similar convention for stateful sets?

Reply:
https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/2217